Makefile

```diff
@@ -266,7 +266,9 @@ clean: ## Clean artifacts
 
 .PHONY: libpodimage
 libpodimage: ## Build the libpod image
-	${CONTAINER_RUNTIME} build -t ${LIBPOD_IMAGE} .
+	-rm -rf $@
+	git clone https://github.com/containers/automation_images.git $@
+	${CONTAINER_RUNTIME} build -t ${LIBPOD_IMAGE} -f podman/Containerfile $@
```

Comment:
Sorry - I'm going to need a whole lot more context to understand this. What is this used for? How is it used? What magic setup is needed? I tried running these commands and just get a failure on STEP 1, almost certainly because I'm not defining `BASE_NAME` and `BASE_TAG`. How does this compensate for that?

I'm also uncertain about the dependency between repos: any change to Containerfile in that repo will have to be sync'ed to this Makefile in the podman repo, and worse, possibly that has to be done on all active maintenance branches. Is there any way to build this image from the `automation_images` repo instead? (Is that what your recent comment suggests?)

I'm also really uncomfortable about running a git checkout within a git checkout. Did you consider:
```make
    LIBPODIMAGE_TMP=$(shell mktemp -d -t $@.tmp.XXXXXXX); git clone ... && ${CONTAINER_RUNTIME} ... $$LIBPODIMAGE_TMP; rm -rf $$LIBPODIMAGE_TMP
```